consensus/service/src/config/tokens.rs

```diff
@@ -521,4 +655,176 @@ mod tests {
             Some(1)
         );
     }
+
+    #[test]
+    fn master_minters_deserialize_serialize_deserialize_works() {
```

Comment:
Nit: This test only does one ser-der round trip, so remove the first `deserialize_`